lib/tree-iterator.js

```diff
@@ -54,7 +54,12 @@ function iterate(node, cb) {
                 XJSOpeningElement: ['name', 'attributes'],
                 XJSAttribute: ['name', 'value'],
                 XJSSpreadAttribute: ['argument'],
-                XJSText: null
+                XJSText: null,
+                // Esprima 2.3 introduced handlers and handler which duplicated
+                // CatchClause nodes. Can't update estraverse until we
+                // replace esprima-harmony-jscs for babel
+                // See https://github.com/jquery/esprima/issues/1031
+                TryStatement: ['block', 'handler', 'finalizer']
```

Comment:
Looks like validateIndentation was the only rule that explicitly used `handlers`, but you also added support for `handler` :) https://github.com/jscs-dev/node-jscs/blob/cbfde5db457ee49673494a6a65f75fea4e049a52/lib/rules/validate-indentation.js#L448

>  Also, wondering if we should backCompat this until 2.0?

You mean to replace `handler` with `handlers`?